python/res/enkf/queue_config.py

```diff
@@ -38,25 +39,42 @@ class QueueConfig(BaseCClass):
     _queue_driver          = ResPrototype("driver_ref queue_config_get_queue_driver(queue_config, char*)")
     _get_num_cpu           = ResPrototype("int queue_config_get_num_cpu(queue_config)")
 
-    def __init__(self, user_config_file=None, config_content=None):
-        if user_config_file is not None and config_content is not None:
-            raise ValueError("Trying to create QueueConfig with multiple configurations!")
+    def __init__(self, user_config_file=None, config_content=None, config_dict=None):
```

Comment:
why not to make as a named dict?

Reply:
That would be a possibility, but this way, we are not breaking the old functionality.

Reply:
Make sense.